documentation/modules/migration-plan-options-ui.adoc

```diff
@@ -8,30 +8,32 @@
 
 On the *Plans for virtualization* page of the {ocp} web console, you can click the {kebab} beside a migration plan to access the following options:
 
-* *Get logs*: Retrieves the logs of a migration. When you click *Get logs*, a confirmation window opens. After you click *Get logs* in the window, wait until *Get logs* changes to *Download logs* and then click the button to download the logs.
-* *Edit*: Edit the details of a migration plan. You cannot edit a migration plan while it is running or after it has completed successfully.
-* *Duplicate*: Create a new migration plan with the same virtual machines (VMs), parameters, mappings, and hooks as an existing plan. You can use this feature for the following tasks:
+* *Edit Plan*: Edit the details of a migration plan. You cannot edit a migration plan while it is running, or after it has been completed successfully.
+* *Start migration*: Active only if relevant.
+* *Restart migration*: Restarts a migration that was interrupted. Before choosing this option, make sure there are no error messages. If there are, you need to edit the plan.
+* *Cutover*: Warm migrations only. Active only if relevant.
+* *Cancel scheduled cutover*: Cancel a scheduled cutover migration for a warm migration plan. Active only if relevant.
+
+* *Duplicate Plan*: Create a new migration plan with the same virtual machines (VMs), parameters, mappings, and hooks as an existing plan. You can use this feature for the following tasks:
 
 ** Migrate VMs to a different namespace.
 ** Edit an archived migration plan.
 ** Edit a migration plan with a different status, for example, failed, canceled, running, critical, or ready.
 
-* *Archive*: Delete the logs, history, and metadata of a migration plan. The plan cannot be edited or restarted. It can only be viewed.
+* *Archive Plan*: Delete the logs, history, and metadata of a migration plan. The plan cannot be edited or restarted. It can only be viewed.
```

Comment:
" *Archive Plan*:  ......The plan cannot be edited or restarted. It can only be viewed". - this is not correct. 
You can edit an archived plan for now. Need to check  if we want to avoid that but currently it's possible.

Reply:
Taking it back, 
The current status is that an archived plan can be partially edited, but this seems wrong. 
We'll fix that as a bug.
So the right behavior is that an archived plan can be only viewed, duplicated or deleted.